pkg/webhook/resources/setting/validator.go

```diff
@@ -146,6 +147,59 @@ func validateVMForceResetPolicy(setting *v1beta1.Setting) error {
 	return nil
 }
 
+// chech if this backup target is updated again by controller to strip secret information
+func (v *settingValidator) isUpdatedS3BackupTarget(target *settings.BackupTarget) bool {
+	if target.SecretAccessKey != "" || target.AccessKeyID != "" {
```

Comment:
return if the type is not `s3`

Reply:
> return if the type is not `s3`

It is a local function, the caller has checked S3, then call here, it is not necessary to check s3 again.

Reply:
updated